OrbitGl/FunctionsDataView.cpp

```diff
@@ -33,14 +33,33 @@ const std::vector<DataView::Column>& FunctionsDataView::GetColumns() {
   return columns;
 }
 
+bool FunctionsDataView::ShowSelectedFunctionIcon(const FunctionInfo& function) {
+  return GOrbitApp->IsFunctionSelected(function);
+}
+
+bool FunctionsDataView::ShowFrameTrackIcon(const FunctionInfo& function) {
+  if (GOrbitApp->IsFrameTrackEnabled(function)) {
+    return true;
+  } else if (GOrbitApp->HasCaptureData()) {
```

Comment:
This is a clang-tidy warning "do not use else after return".

Reply:
Yes, makes sense. Fixed.